### PR TITLE
AHEAD, MRIQC, Brainhack Cloud

### PR DESCRIPTION
**Pull request by smoia**
Hello submitters!

@27-apizzuti @sdres @satra @katjaq @r03ert0 @piloubazin
@jacobsanz97 @mtorabi59
@stebo85 @complexbrains @PeerHerholz @Remi-Gau @SamGuay @likeajumprope 

could you please check that your section is as you expect it?
Brainhack Cloud team, would you be able to provide a .bib reference file, please?

You can find the built pdf [here](https://github.com/ohbm/hackathon2022proceedings/suites/7407158625/artifacts/302009896)

### CODE REVIEW

**Comment by complexbrains** [comment]
I can only see the paragraphs are not spaced between, but I can see it is due to rendering from a doc format, so I guess we will have to accept it. Thank you for the initiative 🚀 .

**Comment by piloubazin** [comment]
Our figure numbers are not correct (6 instead of 4), looking at the code it's likely because we and the open cloud folks both kept the \label{fig:test}. Replacing by a different name in \label{} and \ref{} should fix it. Thanks for your work putting this all together!

**Comment by smoia** [comment]
@complexbrains and @piloubazin, feel free to make the right changes and create a PR to this branch!

File: summaries/ahead-project.tex [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+\printbibliography
+
+
+\begin{figure*}
+	\centering
+	\includegraphics[width=1\textwidth]{AHEAD_Figure_1.png}
+	\caption{A) Neuroglancer visualization, B) section query notebook, C) Cortical ROI and corresponding depth histograms extracted from the different contrasts available.
+	}
+	\label{fig:test}
```
Comment (smoia) [inline summaries/ahead-project.tex]:
```suggestion
	\label{fig:ahead}
```

File: summaries/brainhack-cloud.tex [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+The Brainhack Cloud team plans to run a series of training modules in various Brainhack events throughout the year to reach researchers from various backgrounds and increase their familiarity with the resources provided for the community while providing free and fair access to the computational resources. The training modules will cover how to use and access computing and storage resources (e.g., generating SSH keys), to more advanced levels covering the use of cloud native technology like software containers (e.g., Docker/Singularity), container orchestration (e.g., Kubernetes), object storage (e.g, S3), and infrastructure as code (e.g., Terraform).
+
+\begin{figure}
+    \centering
+    \includegraphics[width=0.5\textwidth]{brainhack_cloud.png}
+    \caption{A team of brainhack volunteers, applied for Oracle Cloud Credits to support open source projects in and around brainhack with powerful cloud resources on the Oracle Cloud: https://brainhack.org/brainhack\_cloud/
+    }
+    \label{fig:test}
```
Comment (smoia) [inline summaries/brainhack-cloud.tex]:
```suggestion
    \label{fig:cloud}
```

File: summaries/ahead-project.tex [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+
+\subsection{Results}
+
+Visualization and annotation of large neuroimaging data sets can be challenging, in particular for collaborative data exploration. Here we tested two different infrastructures: BrainBox \url{https://brainbox.pasteur.fr/}, a web-based visualization and annotation tool for collaborative manual delineation of brain MRI data, see e.g. \parencite{heuer_evolution_2019}, and Dandi Archive \url{https://dandiarchive.org/}, an online repository of microscopy data with links to Neuroglancer \url{https://github.com/google/neuroglancer}. While Brainbox could not handle the high resolution data well, Neuroglancer visualization was successful after conversion to the Zarr microscopy format (fig.~\ref{fig:test}A).
```
Comment (smoia) [inline summaries/ahead-project.tex]:
```suggestion
Visualization and annotation of large neuroimaging data sets can be challenging, in particular for collaborative data exploration. Here we tested two different infrastructures: BrainBox \url{https://brainbox.pasteur.fr/}, a web-based visualization and annotation tool for collaborative manual delineation of brain MRI data, see e.g. \parencite{heuer_evolution_2019}, and Dandi Archive \url{https://dandiarchive.org/}, an online repository of microscopy data with links to Neuroglancer \url{https://github.com/google/neuroglancer}. While Brainbox could not handle the high resolution data well, Neuroglancer visualization was successful after conversion to the Zarr microscopy format (fig.~\ref{fig:ahead}A).
```

File: summaries/ahead-project.tex [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+To help users explore the original high-resolution microscopy sections, we also built a python notebook to automatically query the stains around a given MNI coordinate using the Nighres toolbox~\parencite{huntenburg_nighres_2018} (fig.~\ref{fig:test}B).
```
Comment (smoia) [inline summaries/ahead-project.tex]:
```suggestion
To help users explore the original high-resolution microscopy sections, we also built a python notebook to automatically query the stains around a given MNI coordinate using the Nighres toolbox~\parencite{huntenburg_nighres_2018} (fig.~\ref{fig:ahead}B).
```

File: summaries/ahead-project.tex [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
... we used the MRI-quantitative-R1 contrast to define, segment the region of interest and compute cortical depth measurement. In ITK-SNAP \parencite{Yushkevich2006} we defined the somato-motor area by creating a spherical mask (radius 16.35mm) around the ‘hand knob’ in M1. To improve the intensity homogeneity of the qMRI-R1 images, we ran a bias field correction (N4BiasFieldCorrection, \parencite{Cox1996}). Tissue segmentation was restricted to S1 and was obtained by combining four approaches: (i) fsl-fast \parencite{Smith2004} for initial tissues probability map, (ii) semi-automatic histogram fitting in ITK-SNAP, (iii) Segmentator \parencite{Gulban2018}, and (iv) manual editing. We used the LN2\_LAYERS program from LAYNII open source software \parencite{Huber2021}  to compute the equi-volume cortical depth measurements for the gray matter. Finally, we evaluated cortical depth profiles for three quantitative MRI contrasts (R1, R2, proton density) and three microscopy contrasts (thionin, bieloschowsky, parvalbumin) by computing a voxel-wise 2D histogram of image intensity (fig.~\ref{fig:test}C). Some challenges are indicated by arrows 2 and 3 in the lower part of fig.~\ref{fig:test}C.
```
Comment (smoia) [inline summaries/ahead-project.tex]:
```suggestion
For the cortical profile analysis we restricted our analysis on S1 (BA3b) as a part of the somato-motor area from one hemisphere of an individual human brain. S1 is rather thin (\(\sim\)2mm) and it has a highly myelinated layer 4 (see arrow fig.~\ref{fig:test}C). In a future step, we are aiming to characterize differences between S1 (BA3b) and M1 (BA4). For now, we used the MRI-quantitative-R1 contrast to define, segment the region of interest and compute cortical depth measurement. In ITK-SNAP \parencite{Yushkevich2006} we defined the somato-motor area by creating a spherical mask (radius 16.35mm) around the ‘hand knob’ in M1. To improve the intensity homogeneity of the qMRI-R1 images, we ran a bias field correction (N4BiasFieldCorrection, \parencite{Cox1996}). Tissue segmentation was restricted to S1 and was obtained by combining four approaches: (i) fsl-fast \parencite{Smith2004} for initial tissues probability map, (ii) semi-automatic histogram fitting in ITK-SNAP, (iii) Segmentator \parencite{Gulban2018}, and (iv) manual editing. We used the LN2\_LAYERS program from LAYNII open source software \parencite{Huber2021}  to compute the equi-volume cortical depth measurements for the gray matter. Finally, we evaluated cortical depth profiles for three quantitative MRI contrasts (R1, R2, proton density) and three microscopy contrasts (thionin, bieloschowsky, parvalbumin) by computing a voxel-wise 2D histogram of image intensity (fig.~\ref{fig:ahead}C). Some challenges are indicated by arrows 2 and 3 in the lower part of fig.~\ref{fig:ahead}C.
```

**Comment by stebo85** [comment]
Dear @smoia,

The citations in the brainhack cloud section don't look right:
image

Can you fix this or merge this pull request so we can propose changes?

Also the URLs seem to be too long for the latex template:
image

Is there any trick to break a link over a linebreak?

**Comment by smoia** [comment]
Hello @stebo85 !
I rebased this PR onto main and fixed the references across the two BHC files. You can find the latest version [here](https://github.com/ohbm/hackathon2022proceedings/suites/7434972620/artifacts/303870599)

Do not worry too much about the hyperlinks breaking pages - this is not necessarily the final template, just a working one.

**Pull request merged**